api/tests/test_tasks.py

```diff
@@ -54,30 +86,52 @@ def test_pdf_persistance(self, mock_client):
         order = serializer.validated_data
         create_delivery(order)
 
-        mock_delivery.print_pdf.assert_called_once()
+        mock_delivery.send_mail_to_vendor.assert_called_once()
 
     @patch('api.tasks.Client', autospec=True)
     def test_failure(self, mock_client):
         client = MagicMock()
-        client.create_delivery.return_value = Error({'mensaje': 'ERROR IN THE RECEIVED DATA'})
+        vendor = MagicMock()
+        delivery = Delivery({
+            "respuesta": "102",
+            "mensaje": "ERROR IN THE RECEIVED DATA", "numPedido": "test", "order_id": 33},
+            vendor)
+        client.create_delivery.return_value = [
+            delivery
+        ]
```

Comment:
it's not worth splitting this line. I guess it's a left-over.

Reply:
I guess it was done by my black atom extension, sometimes it does that xD